synapse/util/frozenutils.py

```diff
@@ -66,5 +66,7 @@ def _handle_frozendict(obj):
 
 # A JSONEncoder which is capable of encoding frozendics without barfing
 frozendict_json_encoder = json.JSONEncoder(
+    ensure_ascii=False,
```

Comment:
I don't think this should be necessary (and it makes simplejson much slower). Could you add comments to explain why it is necessary?

Reply:
as discussed, let's just coerce the result to unicode